src/execution/__tests__/variables-test.js

```diff
@@ -98,7 +98,6 @@ const TestType = new GraphQLObjectType({
     }),
     fieldWithNestedInputObject: fieldWithInputArg({
       type: TestNestedInputObject,
-      defaultValue: 'Hello World',
```

Comment:
This actually caused a bunch of failures in the existing suite, so that's promising!